uint16 whichArm
geometry_msgs/Pose pose
bool isGraspable
uint16 id
